esp-hal/src/touch.rs

```diff
@@ -85,12 +85,12 @@ pub struct TouchConfig {
 }
 
 /// This struct marks a successfully initialized touch peripheral
-pub struct Touch<'d, TOUCHMODE: TouchMode, MODE: Mode> {
+pub struct Touch<'d, TOUCHMODE: TouchMode, Dm: Mode> {
```

Comment:
Outside the scope of this PR, but we should probably search for other violations of this rule as well.